app/assets/stylesheets/spaces.scss

```diff
@@ -0,0 +1,358 @@
+// Place all the styles related to the spaces controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+/* IMPORT BOOTSTRAP */
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* VARIABLES */
+
+/* Colors */
+$spaces_blue : rgb(115,210,226);
+$spaces_green: rgb(25,232,141);
```

Comment:
take a look in the vivify repo at `app/assets/` and see how we've structured things there.  The main `application.sass` manifest pulls in sub-stylesheets globally.  For instance we have a `main.sass` that sets global styles around `body` and some very high-level DOM elements.  Then we set up `colors.sass` to define constants, `text.sass` to create text styles, `fonts.sass` to load the font stack, `elements.sass` with some general mixins ... etc etc etc.

SCSS/SASS let's us think much more in terms of modularity than you get to do with straight CSS